app/src/main/java/com/arygm/quickfix/ui/search/SearchOnBoarding.kt

```diff
@@ -0,0 +1,207 @@
+import com.arygm.quickfix.ui.navigation.TopLevelDestinations
+import com.arygm.quickfix.ui.theme.poppinsTypography
+
+@Composable
+fun SearchOnBoarding(navigationActions: NavigationActions, isUser: Boolean) {
+
+  var searchQuery by remember { mutableStateOf("") }
+
+  val searchCategories =
+      listOf(
+          SearchCategory(
+              icon = Icons.Outlined.ImagesearchRoller,
+              title = "Painting",
+              description = "Find skilled painters for residential or commercial projects.",
+              onClick = {
+                // Search Painters
+              }),
+          SearchCategory(
+              icon = Icons.Outlined.Plumbing,
+              title = "Plumbing",
+              description = "Connect with expert plumbers for repairs and installations.",
+              onClick = {
+                // Search Plumbers
+              }),
+          SearchCategory(
+              icon = Icons.Outlined.NaturePeople,
+              title = "Gardening",
+              description = "Hire professional gardeners for landscaping and maintenance.",
+              onClick = {
+                // Search Gardeners
+              }),
+          SearchCategory(
+              icon = Icons.Outlined.ElectricalServices,
+              title = "Electrical Work",
+              description = "Locate certified electricians for safe and efficient service.",
+              onClick = {
+                // Search Electricians
+              }),
+          SearchCategory(
+              icon = Icons.Outlined.Handyman,
+              title = "Handyman Services",
+              description = "Get help with various minor home repairs and tasks.",
+              onClick = {
+                // Search Handyman
+              }),
+          SearchCategory(
+              icon = Icons.Outlined.CleaningServices,
+              title = "Cleaning Services",
+              description = "Book reliable cleaners for home or office maintenance.",
+              onClick = {
+                // Search Cleaners
+              }),
+          SearchCategory(
+              icon = Icons.Outlined.Carpenter,
+              title = "Carpentry",
+              description = "Hire experienced carpenters for woodwork and construction tasks.",
+              onClick = {
+                // Search Carpenters
+              }),
+          SearchCategory(
+              icon = Icons.Outlined.LocalShipping,
+              title = "Moving Services",
+              description =
+                  "Find professional movers to help with local or long-distance relocation tasks.",
+              onClick = {
+                // Search Movers
+              }))
+
+  BoxWithConstraints {
+    val widthRatio = maxWidth / 411
+    val heightRatio = maxHeight / 860
+    val sizeRatio = minOf(widthRatio, heightRatio)
+
+    // Use Scaffold for the layout structure
+    Scaffold(
+        containerColor = colorScheme.background,
+        topBar = {},
```

Comment:
You can just remove this line as topBar isn't a mandatory field in a scaffold

Reply:
Will leave it there when I'll add your component to switch from Search Screen to Announcement Screen